src/stories/components/footer/footer-link.tsx

```diff
@@ -22,9 +22,9 @@ export const FooterLinks = ({ links, styleLinks }: CuTableColumnLinksProps) => {
     {links.map((link, i) => <Box
       key={`link-${i}`}
       sx={{ mr: `${link.spacingsRight ?? 0}px` }}>
-      <CustomPopover
+      {/* <CustomPopover
```

Comment:
Is this removed in a commit after? We shouldn't keep the comment there

Reply:
Yes , the commnet is delete in this commit:
fix-comment